includes/Modules/Idea_Hub.php

```diff
@@ -161,9 +161,65 @@ protected function create_data_request( Data_Request $data ) {
 					return null;
 				};
 			case 'GET:new-ideas':
-				// @TODO implementation
+				// @TODO: Implement this with the real API endpoint.
 				return function() {
-					return null;
+					return array(
+						array(
```

Comment:
I guess we can't load in the fixture file here instead of duplicating it, eh? (I'm assuming it isn't included with the bundled plugin.)

Not a big deal, just sort of a bummer to need to duplicate it! Though I guess the work is already done and it's temporary, so best to leave well-enough alone 🙂

Reply:
We can consider it a motivation to get the real API in place quicker ;)